ethcore/machine/src/tx_filter.rs

```diff
@@ -68,6 +68,11 @@ impl TransactionFilter {
 		)
 	}
 
+	/// Get a reference to the contract address.
+	pub fn contract_address(&self) -> &Address {
```

Comment:
The same here

Reply:
Agreed. This method can be removed as well as `Machine::tx_filter`.